client/src/App.js

```diff
@@ -13,8 +13,8 @@ function App() {
       <GlobalStyles />
       <Main>
         <Switch>
-          <Route exact path="/login">
-            <Login />
+          <Route exact path="/" component={Login}>
+            <Login></Login>
```

Comment:
Now you use `component` and `children` to render the Login. Only one is required.

Reply:
Oh, how could I miss that 🙈thank you for pointing that out!